app/scripts/directives/create/nextSteps.js

```diff
@@ -3,19 +3,21 @@
 angular.module('openshiftConsole').component('nextSteps', {
   controller: [
     'ProcessedTemplateService',
+    'Navigate',
     NextSteps
   ],
   bindings: {
     project: '<',
     projectName: '<',
     loginBaseUrl: '<',
     fromSampleRepo: '<',
-    createdBuildConfig: '<'
+    createdBuildConfig: '<',
+    onContinue: '<'
```

Comment:
We're handling the callback differently here than `onDailogClosed` below. Would be good to be consistent.